src/FixedPairList.cpp

```diff
@@ -274,6 +282,7 @@ namespace espressopp {
       .def("add", pyAdd)
       .def("size", &FixedPairList::size)
       .def("getBonds",  &FixedPairList::getBonds)
+      .def("remove",  &FixedPairList::remove)
```

Comment:
I still think that the name ```remove``` does not represents what that method exactly do. Moreover, ```disconnect/connect``` and ```clear```/```clearAll```/```deleteAll``` are two different things. First is non-destructive whereas second removes information so in principle they should be separated.